storybook/pages/StatusChatInputPage.qml

```diff
@@ -0,0 +1,111 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import Storybook 1.0
+import Models 1.0
+import shared.stores 1.0
+
+import utils 1.0
+import shared.status 1.0
+
+SplitView {
+    id: root
+
+    QtObject {
+        id: globalUtilsMock
+
+        property bool ready: false
+        property var globalUtils: QtObject {
+            property var plainText: function (htmlText) {
+                return htmlText.replace(/(?:<style[^]+?>[^]+?<\/style>|[\n]|<script[^]+?>[^]+?<\/script>|<(?:!|\/?[a-zA-Z]+).*?\/?>)/g,'')
+            }
+            property var isCompressedPubKey: function (publicKey) {
+                return false
+            }
+        }
+        Component.onCompleted: {
+            Utils.globalUtilsInst = globalUtilsMock.globalUtils
+            Global.dragArea = null
+            globalUtilsMock.ready = true
+        }
+    }
+
+    QtObject {
+        id: rootStoreMock
+
+        property bool ready: false
+
+        property ListModel gifColumnA: ListModel {}
+
+        readonly property var formationChars: (["*", "`", "~"])
+
+        function getSelectedTextWithFormationChars(messageInputField) {
```

Comment:
good candidate for further refactor :) Method like that shouldn't be part of store.

Reply:
Agree! It's a helper function used only by StatusChatInput. Should be part of StatusChatInput refactoring.